### ISSUE TITLE
Code coverage doesn't run without parallel attribute set to true

### ISSUE THREAD

**Issue by Raj-Praveen** [kategengler/ember-cli-code-coverage#219]
Hi, 

Ember-cli : 2.12.3,
ember-code-coverage:0.4.1

Code coverage does not generate any reports without setting the "parallel" : true.
```
module.exports = {
  excludes: ['*/mirage/**/*', '*/tests/**/*', '*/vendor/**/*', '*/app/styles/**/*', '*/lib/**/*', '*/assets/**/*', '*/config/**/*'],
  parallel: true,
  reporters: ['html']
}
```
When I set parallel to false, The report is just empty. As I understand this option is useful when you have ember exam installed and you intend to run tests in parallel but I do not have one.

This is a private repo and wouldn't be able to share the code base unfortunately.

Is this a normal behaviour ? Please help me understand.

**Comment by rwwagner90** [comment]
Hi @Raj-Praveen you'll likely want to try the latest beta, which I believe is 1.0.0-beta.8. 0.4.1 is quite old.

**Comment by Raj-Praveen** [comment]
Hi @rwwagner90 , Thanks for your response.

unfortunately I'm not at the comfort of trying the latest beta since we are on an older version of ember and babel. Although the coverage report does get generated as expected, I was curious to know how does it  work only after setting the attribute  ```parallel : true``` when I don't have ember exam configured.

**Comment by rwwagner90** [comment]
@Raj-Praveen you should try the betas. They fix lots of issues. I do not know why that would happen for you, but 0.4.1 was released 2 years ago. Any bug fixes or updates have been released in the various 1.0.0 betas. Please try them.

**Comment by Turbo87** [comment]
closing due to inactivity
